xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/EscrowIT.java

```diff
@@ -66,7 +67,7 @@ public void createAndFinishTimeBasedEscrow() throws JsonRpcClientErrorException,
     EscrowCreate escrowCreate = EscrowCreate.builder()
       .account(senderWallet.classicAddress())
       .sequence(senderAccountInfo.accountData().sequence())
-      .fee(feeResult.drops().openLedgerFee())
+      .fee(XrpCurrencyAmount.ofXrp(new BigDecimal(1)))
```

Comment:
Is this just 1XRP so our transactions always go through?

Reply:
@mukulljangid wdyt? Shouldn't we use feeutils here?

Reply:
Yess! Replacing it